pkg/scheduler/framework/plugins/noderesources/fit.go

```diff
@@ -517,32 +529,43 @@ func fitsRequest(podRequest *preFilterState, nodeInfo *framework.NodeInfo, ignor
 		return insufficientResources
 	}
 
+	nodeCapacity := nodeInfo.Node().Status.Capacity
+
 	if podRequest.MilliCPU > 0 && podRequest.MilliCPU > (nodeInfo.Allocatable.MilliCPU-nodeInfo.Requested.MilliCPU) {
+		unresolvable := podRequest.MilliCPU > nodeCapacity.Cpu().MilliValue()
+
 		insufficientResources = append(insufficientResources, InsufficientResource{
 			ResourceName: v1.ResourceCPU,
 			Reason:       "Insufficient cpu",
 			Requested:    podRequest.MilliCPU,
 			Used:         nodeInfo.Requested.MilliCPU,
 			Capacity:     nodeInfo.Allocatable.MilliCPU,
+			Unresolvable: unresolvable,
```

Comment:
Let's just do this

```suggestion
		insufficientResources = append(insufficientResources, InsufficientResource{
			ResourceName: v1.ResourceCPU,
			Reason:       "Insufficient cpu",
			Requested:    podRequest.MilliCPU,
			Used:         nodeInfo.Requested.MilliCPU,
			Capacity:     nodeInfo.Allocatable.MilliCPU,
			Unresolvable: podRequest.MilliCPU > nodeCapacity.Cpu().MilliValue(),
```

Reply:
Ditto for other resources

Reply:
Got it, let me fix it.

Reply:
Fix for cpu/mem/storage, for extend resource, since `Unresolvable: rQuant > (&nodeCapacity[rName]).Value(),` would raise compile error due to `&` would not work in temp return. So I create `rCapacity` to workaround